definitions/ext-rfscanner/golden_metric.yml

```diff
@@ -0,0 +1,39 @@
+# Determine metrics for VALET
+
+
+latency:
+  title: Latency  (ms)
+  displayAsValue: true
```

Comment:
what makes more sense here? seeing the latency over time or just the last data point?

Reply:
Makes more sense to see it over time. Should we set displayAsValue: false?